stellarphot/core.py

```diff
@@ -353,11 +353,10 @@ class PhotometryData(BaseEnhancedTable):
     In addition to these required columns, the following columns are created based
     on the input data during creation.
 
-    bjd    (only if ra and dec are all real numbers, otherwise set to np.nan)
     night
 
     If these computed columns already exist in `data` class the class
-    will throw an error a ValueError UNLESS`ignore_computed=True`
+    will throw an error a `ValueError` UNLESS ``ignore_computed=True``
```

Comment:
Why the double backticks?

Reply:
I think single backticks is supposed to be a reference and double means format it as code. Not sure why that wasn't an error before, but maybe it was pointing to the attribute?